src/NewToDo/NewToDo.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+import './NewToDo.css';
+
+class NewToDo extends React.Component {
+  state = {
+    userId: 1,
+    title: '',
+    completed: true,
+    user: 'select user',
+    invalidTitle: false,
+    invalidUser: false,
+  };
+
+  changeUser = (event) => {
+    const userName = event.target.value;
+
+    this.setState(prevState => (
+      {
+        user: userName,
+        userId: this.props.users.find(
+          user => user.name === userName,
+        ).id,
+        invalidUser: false,
+      }));
+  };
+
+  changeTextArea = (event) => {
+    this.setState(
+      {
+        title: event.target.value.replace(/\W{2,}/g, ' '),
+        invalidTitle: false,
+      },
+    );
+  };
+
+  isCompletedTask = (event) => {
```

Comment:
this is a name for a variable with boolean, function names are verbs
`checkTask` for example